guide/inheritance-wallet/landing-page.md

```diff
@@ -0,0 +1,208 @@
+---
+layout: guide
+title: Inheritance wallet
+description: An in-depth UX reference design for a multi-key bitcoin wallet with inheritance features designed for families.
+nav_order: 6
+has_children: true
+permalink: /guide/inheritance-wallet/
+main_classes: -no-top-padding
+image: https://bitcoin.design/assets/images/guide/inheritance-wallet/header.png
+image_base: /assets/images/guide/inheritance-wallet/
+images_creation-wallet-basics:
+    - file: 
+      alt: 
+      caption: 
+
+
+---
+
+<!--
+
+Editor's notes
+
+This page covers ....  
```

Comment:
What does it cover? Editor's notes are for people who might edit this file later to better understand editorial decisions made during writing, that might not be obvious.